### DISCUSSION TITLE
Similar estates only allow title picture

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#485] [_Discarded pitches]
### Current state
For similar estates only "title picture" is supported.

e.g. a customer has uploaded a title picture in onOffice enterprise which is only supposed to be used for other real estate portals such as the german portal: Immobilienscout24. It is only possible to chose one "title picture" in onOffice enterprise. In this case the plugin can not use the picture on the website.

Therefore the similar estate card in the estate detail page does not have a image displayed on their website.

### Desired state

All other image types should be supported and able to be used as they are in the estate detail page. Please make them selectable as in the detail page “picture types”.

See attachments (detailpage_pictures.png) and (similarestates_only_titlepic.png)

![detailpage_pictures](https://user-images.githubusercontent.com/88546396/229799618-4db07b03-9f2f-42e9-ab0d-feef4be70f8d.jpg)
![similarestates_only_titlepic](https://user-images.githubusercontent.com/88546396/229799617-cb1ec62d-806a-4765-9ebc-5d729696208e.jpg)

**Comment by fredericalpers** [comment]
@dai-eastgate Please investigate and let me know how long this will approx. take.

**Reply by dai-eastgate** [reply]
@fredericalpers I need 4 days to fix and test it.
